demos/notebooks/demo_pipeline.ipynb

```diff
@@ -570,19 +970,15 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "#%% STOP CLUSTER and clean up log files\n",
-    "cm.stop_server(dview=dview)\n",
-    "log_files = glob.glob('*_LOG_*')\n",
-    "for log_file in log_files:\n",
-    "    os.remove(log_file)"
+    "%%time\n",
+    "cnmf_refit = cnmf_fit.refit(images, dview=multiprocessing_pool)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### View movie with the results\n",
-    "We can inspect the denoised results by reconstructing the movie and playing alongside the original data and the resulting (amplified) residual movie"
+    "The spatial contours of the new estimates should now look cleaner and more \"neuronal\":"
```

Comment:
Yes this is tough as it is basically "use your pattern recognition to see the obvious" kind of thing and it would take a lot of text to actually unpack it. Will think about it.